### PR TITLE
Add docs for Full Observer Node

### PR DESCRIPTION
**Pull request by peterargue**
## Description

Add documentation for the Full Observer Node under Node Operations.

### CODE REVIEW

File: docs/content/node-operation/full-observer-node.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+The node does the following:
+* Maintains state by following a staked access node
+* Verifies and stores blocks received from its upstream staked access node
+* Serves the Access API
+    * Get requests are processed using the node's local state
+    * SendTransaction and script executions are forwarded to the configured staked access node
+* Participates in the LibP2P network for unstaked nodes
+
+The node does not:
+* Sync the full blockchain history. For archival functionality, see [DLP](#data-provisioning-service).
```
Comment (smnzhu) [inline docs/content/node-operation/full-observer-node.mdx:18]: Is this a typo?

Comment (peterargue) [reply docs/content/node-operation/full-observer-node.mdx]: yea it should be `DPS`. I removed that in the latest revision.

**Review by vishalchangrani — APPROVED** [review]
lgtm

**Review by huitseeker — APPROVED** [review]
It's not shakespeare, but it's a start!

**Review by smnzhu — APPROVED** [review]

**Pull request merged**